package.json

```diff
@@ -18,16 +18,15 @@
 		"glob": "7.0.0",
 		"mkdirp": "0.5.1"
 	},
-	"peerDependencies": {
-		"typescript": "^1.6.0"
-	},
 	"devDependencies": {
+	 	"dojo-loader": ">=2.0.0-beta.5",
 		"intern": "~3.0.0",
 		"tsd": "0.6.5",
-		"tslint": "3.5.0"
+		"tslint": "3.11.0",
+    "typescript": "2.0.2"
```

Comment:
tabs vs spaces (fwiw in dojo2 we've standardised on spaces for the `package.json` as that is the format dictated by npm)